src/Microsoft.ML.Core/Data/Schema.cs

```diff
@@ -19,7 +22,7 @@ namespace Microsoft.ML.Runtime.Data
     /// and values.
     /// </summary>
     [System.Diagnostics.DebuggerTypeProxy(typeof(SchemaDebuggerProxy))]
-    public sealed class Schema : ISchema
+    public sealed class Schema : ISchema, IReadOnlyList<Schema.Column>
```

Comment:
>Schema [](start = 24, length = 6)

This file and the new files are significant changes. The others are mechanical, to make the solution build.